iothub_client/tests/common_e2e/iothubclient_common_e2e.c

```diff
@@ -104,6 +104,14 @@ typedef struct EXPECTED_SEND_DATA_TAG
     IOTHUB_MESSAGE_HANDLE msgHandle;
 } EXPECTED_SEND_DATA;
 
+#define MAX_SEND_DATA_BUNDLE_ITEM_COUNT 3500
```

Comment:
Is there a way we could make this to be the same as `TELEMETRY_BATCH_SIZE` so there's one source of truth?